### PR TITLE
Add README best practices

### PR DESCRIPTION
**Pull request by cbeer**

### CODE REVIEW

File: best-practices/howto_readme.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+## Requirements
+
+- Ruby 3.0
+- libxml 5
+
+## Installation
+
+Add this line to your Rails application's Gemfile
+
+\```ruby
```
Comment (cbeer) [inline best-practices/howto_readme.md]: Anyone know how to escape these correctly?

**Comment by mejackreed** [comment]
:+1:

**Comment by ndushay** [comment]
Should this say something explicitly about a README should include the badges and should blah blah blah (it shows by example, but ...)?   Or is it obvious enough as is?

**Comment by azaroth42** [comment]
@ndushay If you think it should, then create a new issue for that to determine consensus.

**Comment by azaroth42** [comment]
:+1:

**Comment by azaroth42** [comment]
@ndushay It does say it should include relevant badges (line 11) and if it didn't that should be a new proposal, rather than blocking the current one.

File: best-practices/howto_readme.md (hunk 1 above)
Comment (darrenleeweber) [inline best-practices/howto_readme.md]: Don't put them in an .md file?  If an example README.md file is required, don't describe it in .md but create it as a template.  Perhaps DLSS best practices can be incorporated into several "template repos", one for each type of project (rails 3, rails 4, sinatra, gem, etc.), e.g.
  - https://github.com/sul-dlss/robot-suite-base

**Comment by ndushay** [comment]
@azaroth42 you are right - i somehow missed line 11.  :+1:

File: best-practices/howto_readme.md (hunk 1 above)
Comment (cbeer) [inline best-practices/howto_readme.md]: @ndushay suggested putting the example in block text and it seems avoid any escaping problems.

I think it's good to provide an example in-line to provide quick and easy access to a model readme. As you can see, I've also linked to a number of README files that are following this practice.

I think having template repositories is an interesting idea, but I'd argue out of scope for this proposal, which only attempted to document the consensus we seemed to have during a F2F meeting.

**Comment by darrenleeweber** [comment]
:+1:

**Pull request merged**